openapi.yaml

```diff
@@ -8578,3 +8626,142 @@ components:
       - title
       - total_stats
       type: object
+    FilterDeploymentsRequest:
+      additionalProperties: false
+      description: Filters to select the deployments in `/filter/deployments`.
+      example:
+        account: 1
+        date_from: 2021-06-23
+        date_to: 2021-12-24
+        in:
+        - github.com/athenianco/athenian-api
+        - github.com/athenianco/athenian-webapp
+        - github.com/athenianco/metadata
+        environments:
+        - production
+        - staging
+        conclusions:
```

Comment:
So if I understand correctly, the API will allow to filter deployments by env and conclusion?

Reply:
Yes. Because it's cheap.